docs/components/alert/README.md

```diff
@@ -66,33 +66,41 @@ CdrAlert is a simple wrapper component that allows for composing various alert l
 
 There are four different options for styling the alert, based on the [alert type](../alert/#guidelines). 
 
-Alerts should be passed an icon component and text
+Alerts should be passed the icon component with the appopriate icon and text for the alert message. 
 
 <cdr-doc-example-code-pair repository-href="/src/components/alert"
 :sandbox-data="$page.frontmatter.sandboxData" >
 
 ```html
 <cdr-alert type="info">
-  <icon-information-fill /> Info Alert
+  <icon-information-fill /> Info alert
 </cdr-alert>
 <br>
 <cdr-alert type="success">
-  <icon-check-fill/> success alert
+  <icon-check-fill/> Success alert
 </cdr-alert>
 <br>
 <cdr-alert type="warning">
-  <icon-warning-fill/> warning alert
+  <icon-warning-fill/> Warning alert
 </cdr-alert>
 <br>
 <cdr-alert type="error">
-  <icon-x-fill/> error alert
+  <icon-x-fill/> Error alert
 </cdr-alert>
 ```
 </cdr-doc-example-code-pair>
 
 
 ## Accessibility
-TODO
+
+Many WCAG requirements are contextual to their implementation. To ensure that usage of this component complies with accessibility guidelines you are responsible for the following:
+- For static messaging that calls out the type of alert in the text and that is loaded with the page there is no need to do anything specific
+- For messages that do not include text to identify what type of alert is being displayed users should add screen reader only text that adds this information to the start of their message
+- Provide a text description when user input falls outside the required format or values: [W3C Techniques for WCAG 2.1](https://www.w3.org/WAI/WCAG21/Techniques/general/G83)
+- Don't rely on color alone to convey your message. Provide an additional indicator to color, like an icon: [WCAG 1.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html)
```

Comment:
like an icon with appropriate screen reader text -- seems like this could be grouped with the "For messages that do not include text to identify what type of alert is being displayed users should add screen reader only text that adds this information to the start of their message"

Reply:
@mhewson are you suggesting that the bullets on line 99 and 100 should be sub-bullets of line 98?